src/routes/docs/products/databases/permissions/+page.markdoc

```diff
@@ -30,4 +30,144 @@ Document level permissions are configured in individual documents.
 
 [Learn more about permissions and roles](/docs/advanced/platform/permissions)
 
+# Example: Allow users to create private documents {% #private-documents %}
```

Comment:
This flow is a little uncommon compared to the rest of our docs. 

Also, we have examples over https://appwrite.io/docs/advanced/platform/permissions#examples. Maybe those can be improved with some additional details from here.

Reply:
I've moved it to the platform/permissions doc as example 3 and linked to it from the database/permission doc.
What do you think?